core/templates/services/svg-sanitizer.service.spec.ts

```diff
@@ -45,25 +45,6 @@ describe('SvgSanitizerService', () => {
     'A5OTAwIiBzdHJva2U9IiMwMDQ0MDAiPjwvcG9seWdvbj4KPC9zdmc+'
   );
 
-  /**
-   * Malicious SVG Decoded
-   * <svg version="1.1" baseProfile="full" xmlns="http://www.w3.org/2000/svg">
-   * <polygon id="triangle"
-   *           points="0,0 0,50 50,0" fill="#009900" stroke="#004400" />
-   * <script type="text/javascript">
-   *   alert('This app is probably vulnerable to XSS attacks!');
-   * </script>
-   * </svg>
-   */
-  const maliciousSvg = (
```

Comment:
Where is the replacement for this test?

Reply:
This test has now been renamed to `isBase64Svg`. We are now sanitizing the SVG before using `bypassSecurityTrustResourceUrl` in `getTrustedSvgResourceUrl` method itself.

Reply:
My question still stands. The old test cases should be part of your new test case suite in some way, so that we don't reduce coverage.

Reply:
I have added this test case,i.e, when the SVG has a hidden script in the next test(line 162). The code coverage for the `svg-sanitizer.service.ts` is 100% and all branches are covered.